spec/services/waste_carriers_engine/convictions_check/person_match_service_spec.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+module WasteCarriersEngine
+  module ConvictionsCheck
+    RSpec.describe PersonMatchService do
+      describe "run" do
+        let(:time) { Time.new(2019, 1, 1) }
+        before { allow(Time).to receive(:current).and_return(time) }
+
+        let(:first_name) { "foo" }
+        let(:last_name) { "bar" }
+        let(:date_of_birth) { Date.today }
+
+        let(:entity_a) do
+          double(:entity,
+                 system_flag: "foo",
+                 incident_number: "bar",
+                 name: "baz")
+        end
+        let(:entity_b) do
+          double(:entity,
+                 system_flag: "qux",
+                 incident_number: "quux",
+                 name: "quuz")
+        end
+
+        let(:subject) do
```

Comment:
Why `let`?
```suggestion
        subject do
```
Or, since the `subject` is actually the `result` of the `run` action, can we call this `result`? It will avoid confusion with the fact that `subject` is expected to contain an instance of the testing class by convention ::)